docs/api-reference/data-fetching/getServerSideProps.md

```diff
@@ -0,0 +1,114 @@
+---
+description: API reference for `getServerSideProps`. Learn how to fetch data on each request with Next.js.
+---
+
+# `getServerSideProps`
+
+<details>
+  <summary><b>Version History</b></summary>
+
+| Version   | Changes                                                             |
+| --------- | ------------------------------------------------------------------- |
+| `v10.0.0` | `locale`, `locales`, `defaultLocale`, and `notFound` options added. |
+| `v9.3.0`  | `getServerSideProps` introduced.                                    |
+
+</details>
+
+When exporting an `async` function called `getServerSideProps` (server-side rendering) from a page, Next.js will pre-render this page on each request using the data returned by `getServerSideProps`. This is useful if you want to fetch data that changes often, and have the page update to show the most current data.
+
+```js
+export async function getServerSideProps(context) {
+  return {
+    props: {}, // will be passed to the page component as props
```

Comment:
So this is only true when in `_app` you do `<Component {...pageProps}/>`, without destructuring `pageProps` in any way. For example:

```js
function MyApp({ Component, pageProps: { foo , ...pageProps } }) {
  return <Component {...pageProps} />
}
```
`foo` won't be in the page component's `props`. Not sure if that is useful, I just wanted to point it out.

Reply:
I don't think that has to be explicitly explained. Something that should be explained in the _app docs probably.